src/components/Documentation/Markdown/index.tsx

```diff
@@ -37,10 +39,24 @@ const Details: React.FC<{
   if (!filteredChildren.length) return null
   if (typeof filteredChildren[0] === 'string') return null
 
-  const text = filteredChildren[0].props.children[0]
+  let details = getDetails()
+  const detailTitle = filteredChildren[0].props.children[0]
+  const detailText = getText(filteredChildren.slice(1))
```

Comment:
we need a better way to determine the specific collapsible. collecting the whole text feels too much, expensive.